_episodes/28.md

```diff
@@ -0,0 +1,257 @@
+up and down along with the autoscaling applied to the segments. It consists of
+a controller that maintains stream metadata and the segment store that works off
+of tier one storage (Apache Bookkeeper) and tier two storage (Object storage).
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/28/pravega-architecture.png"/><br/>
+Pravega Docs: <a href="https://pravega.io/docs/nightly/pravega-concepts/#elastic-streams-auto-scaling">Source</a>.
+</p>
+
+Just like Trino, Pravega also aims to build a rich set of connectors with
+systems that will be the source and sink. This includes a connector used for 
+Trino.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/28/pravega-connectors.png"/><br/>
+Pravega Docs: <a href="https://pravega.io/docs/nightly/pravega-concepts/#elastic-streams-auto-scaling">Source</a>.
+</p>
+
+### Pravega compared to other event streaming platforms.
+
+This chart is very helpful resource to summarize Pravega against other popular 
+streaming platforms. This comes from the Pravega site so be sure to check for
+an up to date list of these features moving forward.
+
+|                                    | Pravega | Kafka | Pulsar |
+|------------------------------------|---------|-------|--------|
+| Transactions                       | ✅ | ✅ | ✅ |
+| Event streams                      | ✅ | ✅ | ✅ |
+| Long-term retention                | ✅ |    | ✅ |
+| Durable by default                 | ✅ |    | ✅ |
+| Auto-scaling                       | ✅ |    |    |
+| Ingestion of large data (video)    | ✅ |    |    |
+| efficient at high partition counts | ✅ |    |    |
+| Consistent state replication       | ✅ |    |    |
+| Key-value tables                   | ✅ |    |    |
+
+Comparison between Pravega, Kafka, and Pulsar: [Source](https://pravega.io)
+
+## Demo of the week: Querying Pravega from Trino
+
+This week, Pravega will be showing us a demonstration from their own [getting-started](https://github.com/pravega/presto-connector/tree/main/getting-started)
+tutorial in the Trino/Presto connector.
+
+## PR of the week: Pravega presto-connector PR 49
+
+This weeks [PR of the week](https://github.com/pravega/presto-connector/pull/49)
+doesn't come from the Trino repository this week but rather the presto-connector
+repository. The Trino portion of the repository was committed by Dell engineer 
+Karan Singh. As it states, this now makes Pravega available from Trino along 
+with the original Presto connector.
+            
+Thanks Karan for adding Trino and Andrew for writing the original Presto-Pravega
+connector!
+
+## Question of the week: What is the point of Trino Forum and what is the relationship to Trino Slack?
+
+Our [question of the week](https://www.trinoforum.org/t/what-is-the-point-of-this-forum-and-what-is-the-relationship-to-trino-slack/28)
+comes from the new Trino Forum by Starburst that Brian and a few others at  have put 
```

Comment:
hm .. maybe you are using it as name and hence always use "Trino Forum" .. up to you

Reply:
by Starburst. Brian and a few other helpers have set this up and started to add content.